templates/home.html

```diff
@@ -1,40 +1,39 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:group {"tagName":"main","style":{"spacing":{"margin":{"top":"var(--wp--preset--spacing--70)","bottom":"var(--wp--preset--spacing--70)"}}},"layout":{"inherit":true}} -->
-<main class="wp-block-group" style="margin-top:var(--wp--preset--spacing--70);margin-bottom:var(--wp--preset--spacing--70)">
+<!-- wp:group {"tagName":"main","style":{"spacing":{"margin":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}}},"layout":{"inherit":true}} -->
+<main class="wp-block-group" style="margin-top:var(--wp--preset--spacing--60);margin-bottom:var(--wp--preset--spacing--60)">
 
 	<!-- wp:image {"align":"wide"} -->
 	<figure class="wp-block-image alignwide"><img alt=""/></figure>
 	<!-- /wp:image -->
-	
-	<!-- wp:heading {"align":"wide","style":{"typography":{"fontWeight":"400"},"spacing":{"margin":{"top":"var(--wp--preset--spacing--70)","bottom":"var(--wp--preset--spacing--70)"}}}} -->
-	<h2 class="alignwide" style="margin-top:var(--wp--preset--spacing--70);margin-bottom:var(--wp--preset--spacing--70);font-weight:400;">Mindblown: a blog about philosophy.</h2>
+
+	<!-- wp:heading {"align":"wide","style":{"spacing":{"margin":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}}}} -->
+	<h2 class="alignwide" style="margin-top:var(--wp--preset--spacing--60);margin-bottom:var(--wp--preset--spacing--60);">Mindblown: a blog about philosophy.</h2>
```

Comment:
Same here, the query and search title have a bottom margin using the 50 preset.
Just asking, because I am not sure that they do need to match...

Reply:
Good spot. I've now matched these spacing values, so they're the same across the templates, but I'm not sure if this is correct. Although I think it feels right for them to be consistent.

@critterverse, do you think these spacing values should be the same across the different templates?

Reply:
@mikachan Good change! I think it works best being consistent across templates.